iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealView.swift

```diff
@@ -199,20 +199,20 @@ private extension AddMealView {
 
 // MARK: - Functions
 private extension AddMealView {
-  
+
   func createMeal() -> Meal {
     Meal(id: "\(sdk.getUUID())",
          name: "\(title)",
          quantity: Int32(quantity),
-         availableFromDate: "\(availableFromDate)",
-         expiryDate: "\(useByDate)",
+         availableFromDate: "\(Int(availableFromDate.timeIntervalSince1970))",
+         expiryDate: "\(Int(useByDate.timeIntervalSince1970))",
```

Comment:
The format of this number by default includes the milliseconds so looks like this: `1607446816.3232`